src/Table/Table.jsx

```diff
@@ -98,7 +98,7 @@ export class Table extends Component {
     const disableSort = displayedData.length <= 1;
 
     let pages = [displayedData];
-    if (paginated) {
+    if (paginated && displayedData.length > 0) {
```

Comment:
the chunking logic only works if there's actually data in the array, otherwise the assumptions that follow it breakdown when it returns an empty array instead of an array of arrays.

lemme see if i can rewrite that logic in a clearer way to avoid confusion for anyone reading through